<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PushButtonOptionWidget</class>
 <widget class="QWidget" name="PushButtonOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nom :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="_objectNameEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="_pushButton">
       <property name="text">
        <string>Click()</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_addConnexionButton">
       <property name="text">
        <string>Ajouter une connexion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="_connexionTable">
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>125</number>
       </attribute>
       <column/>
       <column/>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
